websites/P/PayPal/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "author": {
+    "name": "レム",
+    "id": "225657353696051200"
+  },
+  "contributors": [
+    {
+      "name": "Senko Nyan",
+      "id": "251098700259131404"
+    }
+  ],
```

Comment:
How can a presence have contributors prior to it even being added to the store?
![image](https://user-images.githubusercontent.com/64903135/147700980-c811f2aa-42a5-41c7-aab2-73628a55e708.png)